wagtail/contrib/forms/models.py

```diff
@@ -93,6 +93,8 @@ class AbstractFormField(Orderable):
     field_type = models.CharField(
         verbose_name=_("field type"), max_length=16, choices=FORM_FIELD_CHOICES
     )
+    # field_type must be populated for previews to build the form field.
+    field_type.required_on_save = True
```

Comment:
An alternative is to provide a fallback when `field_type` is an empty string, either here:

https://github.com/wagtail/wagtail/blob/d6bc5f95643e5628a8e42920fbe49210ad34b36f/wagtail/contrib/forms/forms.py#L131-L134

or here: 

https://github.com/wagtail/wagtail/blob/d6bc5f95643e5628a8e42920fbe49210ad34b36f/wagtail/contrib/forms/forms.py#L87-L96

but I'm not sure what's more helpful: to highlight the missing value, or to silently suppress it during previews/draft state.